package.json

```diff
@@ -23,12 +19,8 @@
     "babel-eslint": "^10.0.1",
     "babel-jest": "^23.4.2",
     "babel-loader": "^8.0.4",
-    "babel-plugin-transform-es2015-modules-commonjs": "^6.26.2",
-    "babel-preset-flow": "^6.23.0",
```

Comment:
we use it in babel.config.js

Reply:
I think that one is using `@babel/preset-flow`.

Reply:
oh, you are right. forgot about it